.github/workflows/build-cross.yml

```diff
@@ -90,8 +90,8 @@ jobs:
           ./configure --prefix="$HOME/cross" --target=x86_64-w64-mingw32 \
               TARGET_LIBDIR="$TESTDIR" || res=$?
           if ! [ "$res" = 0 ]; then cat config.log; exit "$res"; fi
-          make crossopt -j$(nproc)
-          make installcross
+          make -j$(nproc) crossopt
+          make -j$(nproc) installcross
```

Comment:
The `install` targets don't have any dependencies - what are they supposed to be doing in parallel?

Reply:
Right.